mentors/java/m.json

```
@@ -78,5 +78,13 @@
   "link_url": null,
   "avatar_url": null,
   "bio": "I've been developing in Java Professionaly for 5 years now. I'm passionate about writing clean code and learning new programming languages."
+ },
+{
+  "github_username": "mbolis",
+  "name": "Marco Bolis",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I've been writing OO Java code for the last ten years or so. I also know a couple other languages and paradigms. I believe one should learn to think their code from many POVs. And dream big."
```

```suggestion
  },
  {
    "github_username": "mbolis",
    "name": "Marco Bolis",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I've been writing OO Java code for the last ten years or so. I also know a couple other languages and paradigms. I believe one should learn to think their code from many POVs. And dream big."
```
Just some minor formatting changes needed.  There are some formatting errors earlier this file that have propagated like a wave. 

Line 89 should end up indented to the same level as line 82.  Line 83-88 should be indented 2 spaces from the `{`.